suite-native/formatters/src/components/TokenToFiatAmountFormatter.tsx

```diff
@@ -32,7 +34,8 @@ export const TokenToFiatAmountFormatter = ({
 
     if (!rates) return <EmptyAmountText />;
 
-    const amount = toFiatCurrency(value.toString(), fiatCurrency.label, rates, 2);
+    const shiftedValue = Number(value) / 10 ** decimals;
```

Comment:
same here, don't use plain values without explanation please. No one will know why number 10 and you will need to study this hard when we debug it in future

Reply:
maybe even better move this to some function outside of component (could be in same file)

Reply:
convert function created https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25